packages/utils/tokens/src/shadow-tokens.ts

```diff
@@ -0,0 +1,23 @@
+import { ColorValue } from 'react-native';
+import { IOperationSet } from '@uifabricshared/foundation-tokens';
+import { ITheme } from '@uifabricshared/theming-ramp';
+import { tokenBuilder } from './tokenBuilder';
+
+export interface IShadowTokens {
+  shadowColor?: ColorValue;
```

Comment:
I think we have two separate colors for shadows with the new tokens - one is for the shadow around the entire component, and the other if for the shadow that's positioned to the lower right of the component

Reply:
I guess this can be applied to both the key and ambient colors so this isn't an issue